### PR TITLE
New Post: 2021 Scroll Survey

### LINKED ISSUE #4515: content: scroll survey announcement [2021-02-05]
* [draft](https://docs.google.com/document/d/1sPFzK4OJtTm_1uF6FeYhiVESqjNVtPrQ1nOtwUVhmSU)
* [thread](https://groups.google.com/a/google.com/g/webdev-content/c/1SKh19HJ5Cc)

### PR DESCRIPTION
**Pull request by argyleink**
Fixes #4515

### CODE REVIEW

File: src/site/content/en/blog/2021-scroll-survey/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/2021-scroll-survey/index.md:3]: (Sorry, I can't remember if I said this already)

This should be a call-to-action
```suggestion
title: Take the 2021 scroll survey to help improve scrolling on the web
```

**Review by kaycebasques — APPROVED** [review]

**Comment by kaycebasques** [comment]
This looks like a duplicate of #4515

**Comment by argyleink** [comment]
not a duplicate? this is a PR and not an issue. reopen @kaycebasques ? this survey is going through final approvals and checks before shipping.

**Comment by mihajlija** [comment]
@argyleink Heya, I'm gonna reopen this. What's the target publish date?

**Comment by samdutton** [comment]
Just waiting on legal sign-off before running pilot. Will confirm in next
couple of days.

**Comment by argyleink** [comment]
this has moved to the next step and will be getting some attention soon 👍🏻

**Pull request merged**
